README.md

```diff
@@ -4,10 +4,39 @@ DotNetTools
 [![Travis build status](https://img.shields.io/travis/aspnet/DotNetTools.svg?label=travis-ci&branch=dev&style=flat-square)](https://travis-ci.org/aspnet/DotNetTools/branches)
 [![AppVeyor build status](https://img.shields.io/appveyor/ci/aspnetci/DotNetTools/dev.svg?label=appveyor&style=flat-square)](https://ci.appveyor.com/project/aspnetci/DotNetTools/branch/dev)
 
-The project contains command-line tools for the .NET Core CLI.
+This project is part of ASP.NET Core. You can find samples, documentation and getting started instructions for ASP.NET Core at the [Home](https://github.com/aspnet/home) repo.
```

Comment:
Yeah, this line is probably outdated now. All of our repos had this in the readme, and all I did here was just rearrange the line. We should update aspnet/home.